package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "@tryfinch/react-connect",
-  "version": "2.8.0",
-  "description": "",
+  "version": "2.8.1",
+  "description": "Finch SDK for embedding Finch Connect in API React Single Page Applications (SPA)",
```

maybe we add some keywords while we're at it?

```suggestion
  "description": "Finch SDK for embedding Finch Connect in API React Single Page Applications (SPA)",
  "keywords": ["finch", "sdk", "payroll", "react"],
```

---

Also added in a homepage value: tryfinch.com